.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "css.validate": false,
+  "stylelint.enable": true
```

Comment:
Install `stylelint` and `stylelint-config-recommended` dependencies and include it into our Git pre-commit hooks.

## Examples

- [Install `stylelint-config-recommended`](https://github.com/zainfathoni/rumahberbagi-app/commit/725b68186e792aa13e18c315371f17c97a925f6e)
- [Configuring Git pre-commit hooks using `lint-staged`](https://github.com/zainfathoni/rumahberbagi-app/commit/419445c7c3a980bf3ad6499b84a1d8d17e6d410b#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R48)

Reply:
Let's work on it separately at #69.